### PR TITLE
the one where the card get a container

### PR DESCRIPTION
**Pull request by sturobson**
The cards don't really have a container for when they're used. This fixes that.

It's more needed because of the resize/zoom on `:hover` effect. But the container design pattern does have a heading, so this adds that.

todo:

- see if we need a 'level of fanciness' classname for the resize/zoom on `:hover` effect

### CODE REVIEW

**Review by khawkins98 — APPROVED** [review]

**Comment by khawkins98** [comment]
Hah, just saw it's draft.

File: components/vf-card/vf-card.config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# The title shown on the component page
-title: Card
+# The card_title shown on the component page
+card_title: Card
```
Comment (khawkins98) [inline components/vf-card/vf-card.config.yml]: I think the `title` replacement on line 1, 2 here are find-replace typos.

**Comment by khawkins98** [comment]
I think this branch might be out of sync with develop, going to try and pull develop into it.

File: components/vf-card/vf-card.config.yml (hunk 1 above)
Comment (sturobson) [reply components/vf-card/vf-card.config.yml]: how embarrassing

Comment (sturobson) [reply components/vf-card/vf-card.config.yml]: fixed in latest commit

**Comment by khawkins98** [comment]
That seems to have gone well. Changed files are now 17 and not 52.

**Comment by khawkins98** [comment]
Added a little @todo on animation. 

But looks good otherwise

**Pull request merged**